pydra/engine/core.py

```diff
@@ -205,9 +206,11 @@ def cache_dir(self):
     @cache_dir.setter
     def cache_dir(self, location):
         if location is not None:
+            self._custom_cache_dir = True
```

Comment:
do you still need the `_custom_cache_dir`?

Reply:
yes, it's to prevent tasks nesting within a workflow's default `/tmp/path` `cache_dir`

Reply:
wouldn't the other clause automatically override the task's `cache_dir` ?